styles/components/_buttons.scss

```diff
@@ -19,6 +19,7 @@
   cursor: pointer;
   font-weight: 700;
   transition: border 0.15s ease;
+  text-align: center;
```

Comment:
We need to be on the look-out in case this one has some side-effects. I didn't see any, but I only did a cursory look through.

Reply:
Is this the intended effect?

![image](https://user-images.githubusercontent.com/1584275/31789215-629b0354-b4df-11e7-89c6-8c7cc0daef63.png)

If we change height to min-height, we'd get this

![image](https://user-images.githubusercontent.com/1584275/31789329-cd6efc94-b4df-11e7-9781-c9c1391b9e5c.png)

But that would be more disruptive to the layout in some places, because the button would push other elements out of the way.

@morebrownies I believe you asked for the text to truncate. That's not ideal for translations. Do you still want that, or would one of the above be ok?